packages/server/graphql/mutations/helpers/notifications/MattermostNotifier.ts

```diff
@@ -178,37 +179,43 @@ const MattermostNotificationHelper: NotificationIntegrationHelper<MattermostNoti
   },
 
   async endMeeting(meeting, team) {
-    const {facilitatorUserId} = meeting
+    const {facilitatorUserId, summary} = meeting
     const {webhookUrl} = notificationChannel
 
     const summaryText = getSummaryText(meeting)
     const meetingUrl = makeAppURL(appOrigin, `meet/${meeting.id}`)
+    const fields: Field[] = [
+      {
+        short: true,
+        title: 'Team',
+        value: team.name
+      },
+      {
+        short: true,
+        title: 'Meeting',
+        value: meeting.name
+      },
+      {
+        short: false,
+        title: 'Stats',
```

Comment:
The title was previously `Summary` but I thought it'd be confusing to have a `Summary` and an `AI Summary` so I changed it to `Stats`